src/main/java/org/datadog/jmxfetch/Instance.java

```diff
@@ -199,6 +208,40 @@ private void loadMetricConfigFiles(AppConfig appConfig, LinkedList<Configuration
         }
     }
 
+    private void loadMetricConfigResources(AppConfig config, LinkedList<Configuration> configurationList) {
```

Comment:
Is it legit to add a unit tests for this method, unless it is covered already?

Reply:
You most probably will have to create a method to get the `inputStream` so this method can be testable as such by creating an object instance and calling this method while providing your inputStream.

Reply:
Ok, I've added a few simple tests. Is this what you were looking for?